include/swift/Option/FrontendOptions.td

```diff
@@ -509,14 +509,19 @@ def dump_interface_hash : Flag<["-"], "dump-interface-hash">,
    HelpText<"Parse input file(s) and dump interface token hash(es)">,
    ModeOpt;
 
+def build_module_from_interface :
+  Flag<["-"], "build-module-from-interface">,
+  HelpText<"Treat the (single) input as a swiftinterface and produce a module">,
```

Comment:
Since we're moving to calling the output of these "compiled modules", should we change this to "-compile-module-from-interface" while here?